elections/election-2022.md

```diff
@@ -0,0 +1,102 @@
+# Project nomination for BSC representatves 2022
+
+In accordance with the [Elections Schedule](./schedule.md) and the
+[eBPF Foundation Charter][charter], this page holds a public record of the
+steps undertaken to nominate members for the BSC in 2022.
+
+The following members' terms will expire in 2022. The Section 4.b column
+indicates which projects must be considered to draw nominations from.
+
+| Name    | Section 4.b  |
+| ------- | ------------ |
+| (empty) | eBPF runtime |
+| TBD     |              |
+
+## Nominations (ETA August)
+
+The following nominees are under consideration for election to the BSC.
+
+### Kernel Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.i Two Linux Kernel representatives
+from the group of eBPF Linux kernel maintainers (as specified in the Linux eBPF
+maintainer file). The Linux Kernel representatives must not be employed by the
+same Company or by Related Companies._
+
+* TBD
+
+### Runtime Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.ii Up to two representatives from
+other open source eBPF runtime implementations.￼Each “eBPF Runtime
+Representative” will represent a different eBPF runtime chosen by the BSC, and
+must be an active contributor to that eBPF runtime._
+
+* eBPF Runtime for Windows
```

Comment:
From https://ebpf.io/infrastructure, it looks like hBPF might be in this category (i.e., should hBPF move up to this list instead of line 71)?   Also not currently on the ebpf.io page, but one might argue that https://github.com/generic-ebpf/generic-ebpf would be in this category too, just ranked lower in terms of github stars?

Reply:
Makes sense in general. In reality though we may struggle to find an active contributor to the generic-ebpf runtime given that there has been barely any change in the project in the last ~2.5 years.

Reply:
> Makes sense in general. In reality though we may struggle to find an active contributor to the generic-ebpf runtime given that there has been barely any change in the project in the last ~2.5 years.

Agree.  Just because a project is on the eligibility list doesn't mean they actually get nominated or chosen.